index.html

```diff
@@ -0,0 +1,165 @@
+                <!-- Toolbar Name -->
+                <div class="" style="width: 20%;">
+                  <a href="./index.html"><img style="height: 60px; width: 200px;" src="./assets/LogoTWOH.PNG" /></a>
+                </div>
+                <!-- Toolbar Menu -->
+                <nav class="" style="width: 65%; padding-left: 20%; display: flex; flex-direction: row; justify-content: space-between; font-size: 0.8rem;">
+                  <a href="./index.html" style="color: #272a31;"><b>Home</b></a>
+                  <a href="./about.html" style="color: #272a31;"><b>About</b></a>
+                  <a href="./index.html" style="color: #272a31;"><b>Attend</b></a>
+                  <a href="./index.html" style="color: #272a31;"><b>Donate</b></a>
+                  <a href="./index.html" style="color: #272a31;"><b>News</b></a>
+                </nav>
+                <div style="width: 15%; display: flex; flex-direction: row; justify-content: flex-end;"><button style="border: 4px solid #ec5242; outline: none; background: #ffffff;" class="p-2">
+                    <a href="#contact" style="color: #ec5242;"><b>Donate</b></a>
+                </button></div>
+              </div>
+          </header>
+        <div id="menu" style="width: 100vw; height: 100vh; background-color: #272a31; position: fixed; top: 0; left: 0; right: 0; bottom: 0; z-index: 10; display: none;">
+            <div class="">
+                <a id="close-menu" style="display: block; color: #ec5242; font-size: 3rem; position: relative; left: 90%;"><b>&times;</b></a>
+                <a href="./index.html" class="mt-5" style="display: block; color: #ec5242; text-decoration: none; font-size: 1.8rem; position: relative; left: 10%;"><b>Home</b></a>
+                <hr style="width: 100%; background-color: #d3d3d3;">
+                      <a href="./about.html" style="display: block; color: #ec5242; text-decoration: none; font-size: 1.8rem; position: relative; left: 10%; "><b>About</b></a>
+                      <hr style="width: 100%; background-color: #d3d3d3;">
+                      <a href="./index.html" style="display: block; color: #ec5242; text-decoration: none; font-size: 1.8rem; position: relative; left: 10%;"><b>Attend</b></a>
+                      <hr style="width: 100%; background-color: #d3d3d3;">
+                      <a href="./index.html" style="display: block; color: #ec5242; text-decoration: none; font-size: 1.8rem; position: relative; left: 10%;"><b>Donate</b></a>
+                      <hr style="width: 100%; background-color: #d3d3d3;">
+                      <a href="./index.html" style="display: block; color: #ec5242; text-decoration: none; font-size: 1.8rem; position: relative; left: 10%;"><b>News</b></a>
+                      <hr style="width: 100%; background-color: #d3d3d3;">
+              </div>
+          
+        </div>
+        <!-- Main Hero Section with Hamburger Menu -->
+        <main id="hero-section" class="highlevel-section">
```

Comment:
- [ ] Good job so far 👍👍 However, I noticed a grey area which was not present before, and the previous reviewer didn't ask for this. Also, this is not present according to the design guidelines. Please consider fixing this one ⚒ See the images below for more clarity 💡

Your design now 🔽 | Your design previously 🔽
---| ---|
![image](https://user-images.githubusercontent.com/106119105/233698265-7c0b65c8-7e0c-4d8b-9150-88635f66edbf.png) | ![image](https://user-images.githubusercontent.com/106119105/233698337-d05ca8f0-c89e-4583-8996-f56a153533bd.png)

Reply:
Fixed it.